src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -556,5 +556,29 @@ public static PoissonRegression PoissonRegression(this RegressionCatalog.Regress
 
             return new LinearSvmTrainer(CatalogUtils.GetEnvironment(catalog), options);
         }
+
+        /// <summary>
+        /// Predict a target using a random binary classification model <see cref="RandomTrainer"/>.
+        /// </summary>
+        /// <param name="catalog">The <see cref="BinaryClassificationCatalog"/>.</param>
+        public static RandomTrainer Random(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog)
+        {
+            Contracts.CheckValue(catalog, nameof(catalog));
+            return new RandomTrainer(CatalogUtils.GetEnvironment(catalog), new RandomTrainer.Options());
+        }
+
+        /// <summary>
+        /// Predict a target using a binary classification model trained with <see cref="PriorTrainer"/> trainer.
+        /// </summary>
+        /// <param name="catalog">The <see cref="BinaryClassificationCatalog"/>.</param>
+        /// <param name="labelColumn">The name of the label column. </param>
+        /// <param name="weightsColumn">The optional name of the weights column.</param>
+        public static PriorTrainer Prior(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
```

Comment:
>Prior [](start = 35, length = 5)

any samples we can link to ?

Reply:
I'll make a quick one for this and the random trainer. 

---
In reply to: [256072896](https://github.com/dotnet/machinelearning/pull/2510#discussion_r256072896) [](ancestors = 256072896)